Python/bytecodes.c

```diff
@@ -2955,18 +2954,35 @@ dummy_func(
             PyCodeObject *code = (PyCodeObject *)func->func_code;
             DEOPT_IF(code->co_argcount != argcount, CALL);
             DEOPT_IF(!_PyThreadState_HasStackSpace(tstate, code->co_framesize), CALL);
+        }
+
+        op(_INIT_CALL_PY_EXACT_ARGS, (method, callable, args[oparg] -- new_frame: _PyInterpreterFrame*)) {
+            int is_meth = method != NULL;
+            int argcount = oparg;
+            if (is_meth) {
+                callable = method;
+                args--;
+                argcount++;
+            }
             STAT_INC(CALL, hit);
```

Comment:
For my own understanding: this is staying here, because if by some miracle to remove the `_CHECK_CALL_PY_EXACT_ARGS` uop, our stats will still be correct?

Reply:
Hm, I suppose. It just felt to belong in the action rather than the guard.